config/ext_jenkins-infra.yaml

```
@@ -491,6 +491,17 @@ controller:
                       credentialsId('github-app-infra')
 
                       traits {
+                        gitHubSCMSourceStatusChecksTrait {
+                            // Note: changing this name might have impact on github branch protections if they specify status names
+                            name('[infra.ci.jenkins.io] ' + config.name)
+                            skip(false)
+                            // If this option is checked, the notifications sent by the GitHub Branch Source Plugin will be disabled.
+                            skipNotifications(false)
```

isn't this what we want?
```suggestion
                            skipNotifications(true)
```

---

We understood that the notification is the status of the build (started, failed, succeeded) to have the red/yellow/green icon so we stil lwant that.


It's the `suppressLogs(true)` that we want so that we keep a feedback (to help public contributors) with only the stage name / sh command / exit status.

Note that we also tried with `skip(true)` : in that case the pipeline exit status is still reported in the PR conversation tab, with the red/yellow/green icon, and the github checks is set to zero with a direct link to the Jenkins build. Safer but less easy for external contributor.

WDYT?

---

ah right i misread